### DISCUSSION TITLE
Please format report values with comma separators.

### DISCUSSION THREAD

**Discussion by clinthart2003** [locustio/locust#2942] [Ideas]
When reporting large values it would be nice if the webUI and the console summary would format values using comma separators.   We often have billions of requests and it would make reading these values much easier if the values were formatted with comma separators.   
![Screenshot 2024-10-16 at 9 35 05 AM](https://github.com/user-attachments/assets/66961015-a913-4677-9570-12933da8c513)
![Screenshot 2024-10-16 at 9 36 03 AM](https://github.com/user-attachments/assets/ee407276-aa91-443e-b3e5-bfcbdc6d8d0a)

**Comment by andrewbaldwin44** [comment]
Hey @clinthart2003, sorry for the slow response. I would suggest opening this as a feature request and someone might be willing to work on it
